src/wifi.rs

```diff
@@ -828,7 +851,7 @@ impl Drop for EspWifi {
     fn drop(&mut self) {
         unsafe {
             TAKEN.lock(|taken| {
-                self.clear_all().unwrap();
+                // self.clear_all().unwrap();
```

Comment:
Forgotten debugging code?

Reply:
most likely, I think I ran into a crash related to this, but I don't remember for sure